scarb/src/core/lockfile.rs

```diff
@@ -0,0 +1,251 @@
+#![allow(dead_code)]
+
+use anyhow::Result;
+use itertools::Itertools;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeSet;
+use toml_edit::{Array, Document, Item, Value};
+
+use crate::core::{PackageId, PackageName, Resolve};
+
+#[derive(Default, PartialEq, Eq, Clone, Copy, Debug, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum LockVersion {
+    #[serde(rename = "1")]
+    #[default]
+    V1 = 1,
```

Comment:
Can we make this serialize to a number instead of `“1”`? That was a big mistake of the logic we did in Sierra JSON format.